src/Models/HydrostaticFreeSurfaceModels/mg_implicit_free_surface_solver.jl

```diff
@@ -0,0 +1,114 @@
+using Oceananigans.Solvers
+using Oceananigans.Operators
+using Oceananigans.ImmersedBoundaries: ImmersedBoundaryGrid, GridFittedBottom
+using Oceananigans.Architectures
+using Oceananigans.Grids: with_halo, isrectilinear
+using Oceananigans.Fields: Field, ZReducedField
+using Oceananigans.Architectures: device
+using Oceananigans.Models.HydrostaticFreeSurfaceModels: implicit_free_surface_linear_operation!
+
+import Oceananigans.Solvers: solve!, precondition!
+import Oceananigans.Architectures: architecture
+import Oceananigans.Solvers: fill_matrix_elements!
```

Comment:
```suggestion
using Oceananigans.Solvers: fill_matrix_elements!

import Oceananigans.Solvers: solve!, precondition!
import Oceananigans.Architectures: architecture
```

Reply:
If we aren't adding a new method then `using` is more appropriate.

https://docs.julialang.org/en/v1/manual/modules/#Standalone-using-and-import